WISE-Community/WISE
# Removed Legacy Footer Link

Removed the href tag to the Legacy Page

To test
1. Open the wise site on localhost and verify that Legacy is  removed in Footer

> ...  Privacy + Use | Contact Us | Legacy

closes #2261
